src/components/Things.ts

```diff
@@ -709,6 +708,12 @@ export class Things<TGameStartr extends FullScreenPokemon> extends GameStartrThi
      * @param defaults   The default settings for the Thing's class.
      * @remarks This is generally called as the onMake call in an ObjectMakr.
      */
+
+    /**
+     * Initalizing new instance of name class
```

Comment:
This comment describes what's happening, but that's not useful for someone trying to learn what `names` is. Try using the same comment that describes the class (see below).